src/helpers/data.ts

```diff
@@ -20,19 +21,22 @@ export function endpoint(sheet_id: string, page_id = 1): string {
 }
 
 export const days = [
-  'Sunday',
-  'Monday',
-  'Tuesday',
-  'Wednesday',
-  'Thursday',
-  'Friday',
-  'Saturday'
+  'domenica',
+  'lunedì',
+  'martedì',
+  'mercoledì',
+  'giovedì',
+  'venerdì',
+  'sabato'
 ];
 
 export const meetingsPerPage = 10;
 
 //parse google spreadsheet data into state object (runs once on init)
 export function load(data: any): State {
+
+  moment.locale('it'); // Set locale to Italian
```

Comment:
I am not sure if this is the right place to set the locale. My spreadsheet days are in Italian, and it also outputs Italian days on the site.